### PR TITLE
Add rabbitmq canned dash

### PR DESCRIPTION
**Pull request by jackzampolin**
Self explainitory

### CODE REVIEW

File: canned/rabbitmq.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+							"wheres": []
+						}
+					]
+				},
+				{
+					"x": 0,
+					"y": 0,
+					"w": 4,
+					"h": 4,
+					"i": "c46351c6-b33a-4dc2-a053-3517e6c8098e",
```
Comment (goller) [inline canned/rabbitmq.json]: Sadly, all these "i" fields need to be unique ids.  I have a script in the canned directory called uuid.sh.  Run that to get a few of 'em.

I'd like to see "i" removed in the future, but, for now you'll need to ad that.

Comment (jackzampolin) [inline canned/rabbitmq.json]: I changed 1 char in each of the strings 🙊. I'll go back and make actual UUIDs.

Comment (goller) [inline canned/rabbitmq.json]: @jackzampolin Ah, no need.

**Comment by goller** [comment]
@jackzampolin would you add rabbitmq to the README.md?

**Comment by jackzampolin** [comment]
@goller good on green!

**Comment by goller** [comment]
oh... and would add it to CHANGELOG as well?

**Pull request merged**